### DISCUSSION TITLE
[Feature Request] Please Add Swipe Right and left To navigate between Tabs

### DISCUSSION THREAD

**Discussion by Jagatpratap** [heroui-inc/heroui#4663] [Feature requests]
### Is your feature request related to a problem? Please describe.

Unable to swipe Left or Right

### Describe the solution you'd like

Unable to swipe Left or Right

### Describe alternatives you've considered

Unable to swipe Left or Right

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-604/[feature-request]-please-add-swipe-right-and-left-to-navigate-between">ENG-604 [Feature Request] Please Add Swipe Right and left To navigate between Tabs</a></p>

**Comment by AdarshHatkar** [comment]
yes,  we need this

**Comment by rockey2020** [comment]
yes i need this

**Comment by abhinav700** [comment]
@wingkwong can I work on this?
